mmjstool/src/i18n_handlers.js

```diff
@@ -199,3 +199,98 @@ export function i18nSplit(argv) {
         }
     });
 }
+
+export function i18nClean(argv) {
+    const wCode = i18nCleanWebapp(argv);
+    const mCode = i18nCleanMobile(argv);
+    process.exit(wCode || mCode);
+}
+
+export function i18nCleanWebapp(argv) {
+    const webappDir = argv['webapp-dir'];
+    const fPath = path.join(webappDir, 'i18n');
+    return clean(argv, fPath);
+}
+
+export function i18nCleanMobile(argv) {
+    const mobileDir = argv['mobile-dir'];
+    const fPath = path.join(mobileDir, 'assets', 'base', 'i18n');
+    return clean(argv, fPath);
+}
+
+function clean(argv, fPath) {
+    const file = argv.file;
+    const dryRun = argv['dry-run'];
+    const check = argv.check;
+    const r = removeItems(fPath, file, dryRun);
```

Comment:
I don't love single character variables, why not use `error` here instead of `r`? I think it express better what it is.

Reply:
It is actually not an error per se, IMO it is more of a result. Would result work for you?

Reply:
Yes, result is a bit generic for my taste, but would work. I think that are "errors" because is failing if that is there and you the check variables. Maybe `warning` to replace `r` and `warnings` to replace `rs`?